backend/workflow_manager/workflow_v2/execution.py

```diff
@@ -305,6 +306,24 @@ def publish_final_workflow_logs(
             f"{successful_files} successfully executed and {failed_files} error(s)"
         )
 
+    def publish_average_cost_log(self, execution_id, total_files):
+
+        total_cost = round(UsageHelper.get_aggregated_cost(execution_id), 5)
+        average_cost = total_cost / total_files
+        self.publish_log(
+            message=(
+                f"The average cost per file for execution {execution_id} "
+                f"is ${average_cost}"
```

Comment:
```suggestion
                f"The average cost per file for execution '{execution_id}' "
                f"is '${average_cost}'"
```
@pk-zipstack are we sure about cost being reported and computed in dollars? Please check with @hari-kuriakose if we need to convert to euros based on the region being deployed

Reply:
Sure @chandrasekharan-zipstack